examples/lang/syntaxerror.mcl.txt

```diff
@@ -0,0 +1,4 @@
+# this file generates a syntax error, used in lib/cli_test.go
+file "/tmp/mgmt" {
+  content => "test"  # missing trailing comma
```

Comment:
tab for indentation, unless your point was that spaces should be a compile error ;)

Reply:
That would be a good point :)